sw/device/lib/base/BUILD

```diff
@@ -113,6 +113,27 @@ opentitan_functest(
     ],
 )
 
+cc_library(
+    name = "memory_prefixed",
+    srcs = ["memory.c"],
+    hdrs = ["memory.h"],
+    defines = [
+        "OT_PREFIX_MEM_FUNCTIONS",
+    ],
+    deps = [
+        ":macros",
+    ],
+)
```

Comment:
Can't we use `dual_cc_library()` for this purpose. Please see my comment about the macro before thinking about the `OT_PREFIX_MEM_FUNCTIONS` `define`.

Reply:
I think this comment is no longer applicable now that I took your suggestion to use the C macro `OT_PLATFORM_RV32`. Now we can use the same `:memory` target on device and host.

Reply:
Sounds good (as long as we don't have a mock for this).